src/EditorFeatures/CSharpTest/RemoveUnusedMembers/RemoveUnusedMembersTests.cs

```diff
@@ -1604,48 +1603,55 @@ public async Task MultipleFields_SomeUnused()
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedMembers)]
         public async Task MultipleFields_SomeUnused_02()
         {
-            await TestDiagnosticMissingAsync(
+            await VerifyCS.VerifyCodeFixAsync(
 @"class MyClass
 {
-    private int [|_goo|] = 0, _bar = 0;
```

Comment:
Interesting, so we cannot force the validation to a specific span within the document?

Reply:
No, and that's actually a desirable characteristic of the framework. The feature I have considered adding is the ability to run just one specific fix, as opposed to the iterative process used by default.

Reply:
Sounds good